src/sdks/tableau/apis/metadataApi.ts

```diff
@@ -36,7 +36,7 @@ const graphqlResponse = z.object({
             isAutoGenerated: z.boolean().optional(),
             hasUserReference: z.boolean().optional(),
             // BinField specific properties
-            binSize: z.number().optional(),
+            binSize: z.number().optional().or(z.null()),
```

Comment:
Driveby fix here. Happened to notice `binSize` was `null` for one of my datasources.